internal/broadcasters_test.go

```diff
@@ -83,7 +84,7 @@ func testBroadcasterGenerically[V any](t *testing.T, broadcasterFactory func() *
 	})
 }
 
-func TestBroadcasterDataRace(t *testing.T) {
+func TestBroadcasterDataRaceSelf(t *testing.T) {
```

Comment:
This test actually already launches all of the functions in parallel as well. Both in parallel with themselves and with each other. I figure the new test is redundant with this one?

Reply:
You're right. I'm not sure how much value add there is to explicitly running them in random order like the new test, because perhaps the go-routine scheduler already does that.

Seems fine to err on the side of >= 0 more coverage though.